roles/rundeck/tasks/rundeck-cli.yaml

```diff
@@ -0,0 +1,21 @@
+---
+- name: Download the Rundeck CLI inside the container
+  ansible.builtin.shell: podman exec '{{ rundeck_container_name }}' wget {{ rundeck_cli_url }}
+
+- name: Unpack the Rundeck CLI inside the container
+  ansible.builtin.shell: podman exec '{{ rundeck_container_name }}' sudo unzip -qqo {{ rundeck_cli_url | basename }} -d {{ rundeck_cli_container_path }}
+
+- name: Create symlink to Rundeck CLI inside the container
+  ansible.builtin.shell: podman exec '{{ rundeck_container_name }}' sudo ln -sf {{ rundeck_cli_container_path }}/{{ rundeck_cli_url | basename | splitext | first }}/bin/rd {{ rundeck_cli_container_path }}/rd
```

Comment:
sudo ?????

Reply:
Product of getting the zip, unzip it to `/usr/local/bin` then symlink it to always point to the version we specify. DEB or mounting may solve this

Reply:
It is extremely disappointing that the container has `sudo` and states:
```
To run a command as administrator (user "root"), use "sudo <command>".
See "man sudo_root" for details.
``` 
For a start, you just add `--user root` when `exec` in and you change from the default user anyway.